AVsitter2/Plugins/AVprop/[AV]menu.lsl

```diff
@@ -32,6 +32,7 @@ key notecard_query;
 list MENU_LIST;
 list DATA_LIST;
 integer MTYPE;
+integer LMSOURCE; //lmsource self = -1, lmsource linkset = -2
```

Comment:
It's not initialized to any of the valid values here, therefore if the notecard has no LMSOURCE command, it won't have a valid value. Anyway, see the note on the possible values below, which would mean this needs no change other than the comment.